File: official/docs/c#/CarrierAccounts/UpdateCarrierAccount.cs
using EasyPost;

EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

CarrierAccount ca = await CarrierAccount.Retrieve("ca_...");

Dictionary<string, object> credentials = new Dictionary<string, object>() {
  { "pickup_id", "abc123" },
};

await ca.Update(new Dictionary<string, object>() {
  { "credentials", credentials },
});

File: official/docs/java/carrierAccounts/UpdateCarrierAccount.java
import com.easypost.EasyPost;
import java.util.HashMap;

public class UpdateCarrierAccount {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        CarrierAccount carrierAccount = CarrierAccount.retrieve("ca_...");

        HashMap<String, Object> credentials = new HashMap<String, Object>();
        credentials.put("pickup_id", "abc123");

        HashMap<String, Object> params = new HashMap<String, Object>();
        params.put("credentials", credentials);

        carrierAccount.update(params);

        System.out.println(carrierAccount);
    }
}
